news/schema.rst

```diff
@@ -0,0 +1,16 @@
+**Added:**
+
+* Schemas for the tables
```

Comment:
This is not particularly informative.  Can you please expand on it. Also, aren't we calling them collections, not tables?